proto/prysm/v1alpha1/beacon_chain.proto

```diff
@@ -428,6 +428,8 @@ message BeaconBlockContainer {
 
         // Representing a blinded capella block.
         SignedBlindedBeaconBlockCapella blinded_capella_block = 8;
+
+        SignedBeaconBlockDeneb deneb = 9;
```

Comment:
Why not here a blinded version that is already being defined in this PR?

Reply:
13b966725fb78420f8d2a70412952d365942a15e